### DISCUSSION TITLE
Pir moving sensor for Home Assistant

### DISCUSSION THREAD

**Discussion by DukeNuken** [Koenkk/zigbee2mqtt#9353] [Devices]
Hi guys

I am using Aqara moving sensors with Home Assistant with zigbee2mqtt addon which turn on/off the light and it is working fine when it that sensor have a “big” movement, but when you put it in the toilet or bathroom sometimes it does not pick up movement and light turn off. I think the best option here get a more sensitive ZigBee sensor or sensor that supports PIR (Passive infrared sensor) detection.

[https://community-assets.home-assistant.io/original/3X/8/5/85e1a420deb795b755ff146c8641fc7803f8348a.jpeg](url)

So the question is can someone recommend sensors which more sensitive than default Xiaomi/aqara sensors?

**Comment by barcelona92** [comment]
Hi

The Philips HUE Pir sensor is ok function for this
